.github/workflows/test.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        goVer: ['1.16', '1.17', '1.18', '1.19', '1.20', '1.21', '1.22']
+        goVer: ['1.18', '1.19', '1.20', '1.21', '1.22']
```

Comment:
the latest code does not support 1.16 and 1.17?

Reply:
emmmm, yes

Reply:
At this stage, I do not recommend making major incompatible changes in the version. That would make it unavailability for users who is using go <1.18.

Reply:
I can't find the stats data that show the users' Go versions of a Go package.
But I doubt the percentage of users with Go versions < 1.18.
We should gather some data insights and make decisions based on the usage stats.

Reply:
As I mentioned in #322 

According to https://go.dev/doc/devel/release

```
Each major Go release is supported until there are two newer major releases. For example, Go 1.5 was supported until the Go 1.7 release, and Go 1.6 was supported until the Go 1.8 release. We fix critical problems, including [critical security problems](https://go.dev/security), in supported releases as needed by issuing minor revisions (for example, Go 1.6.1, Go 1.6.2, and so on).
```

The Go team will not fix critical security problems for versions < 1.20 now.

Reply:
https://telemetry.go.dev/#charts%3agolang.org%2fx%2ftools%2fgopls%3aGoVersion

This result is meaningful for the above discussion.